src/Void.UI/Program.cs

```diff
@@ -10,12 +10,12 @@ namespace Void.UI
     static class Program
     {
         [STAThread]
-        static void Main()
+        static void Main(string[] args)
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
             var inputModeChanger = new WinFormsInputModeChanger();
-            Bus bus = Init.buildVoid(inputModeChanger);
+            Bus bus = Init.buildVoid(inputModeChanger, Options.parse(args));

```

Comment:
I'm okay with merging this, but long-term I have a little different vision for command-line arguments. It may or may not work out, and it would probably be a premature abstraction to introduce at this point, but as soon as we add any more command-line arguments, we need to revisit it.

My long-term vision is something more like this: the last thing that `Main(string[] args)` does before it starts the main GUI loop is to fire off some event, like `CommandLineArgumentsProvided`, which would have the raw, unparsed arguments. Then some command line argument parser service could pick those up and parse them and publish them out as other messages, with would result in the appropriate effects.

But not wanting to jump to a premature abstraction, this is fine for now.